### ISSUE TITLE
null context on getResources

### ISSUE THREAD

**Issue by kakarrot** [crisp-im/crisp-sdk-android#116]
## Crash log ##
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.res.Resources android.content.Context.getResources()' on a null object reference
       at im.crisp.client.internal.c.b.c(b.java:23)
       at im.crisp.client.internal.f.b.a(b.java:293)
       at im.crisp.client.internal.f.b.a(b.java:1)
       at im.crisp.client.internal.f.a.$r8$lambda$0twJuw9WgM4yewfdsLTkXu5dkVo(a.java:27)
       at im.crisp.client.internal.f.a.$r8$lambda$0twJuw9WgM4yewfdsLTkXu5dkVo(a.java:21)
       at im.crisp.client.internal.f.a.$r8$lambda$7WH2Gq-4pww6FTnjmzNmv736F-s(a.java:226)
       at im.crisp.client.internal.f.a.l(a.java:18)
       at im.crisp.client.internal.f.a.$r8$lambda$hUjORiLNZp9Ocd3jUVRv0imbmus(a.java)
       at im.crisp.client.internal.f.a$$InternalSyntheticLambda$0$b65b69a395d3b30551e5f1188667f3dfb0fdfb04a3569291f5654fbed7edda4b$11.call$bridge(a.java:14)
       at io.socket.emitter.Emitter.emit(Emitter.java:26)
       at io.socket.client.Socket.onevent(Socket.java:2)
       at io.socket.client.Socket.onpacket(Socket.java:1)
       at io.socket.client.Socket$2$2.call(Socket.java:9)
       at io.socket.emitter.Emitter.emit(Emitter.java:26)
       at io.socket.client.Manager.ondecoded(Manager.java:2)
       at io.socket.client.Manager$5.call(Manager.java:2)
       at io.socket.parser.IOParser$Decoder.add(IOParser.java:8)
       at io.socket.client.Manager$2.call(Manager.java:2)
       at io.socket.emitter.Emitter.emit(Emitter.java:26)
       at io.socket.engineio.client.Socket.onPacket(Socket.java:1)
       at io.socket.engineio.client.Socket$5.call(Socket.java:1)
       at io.socket.emitter.Emitter.emit(Emitter.java:26)
       at io.socket.engineio.client.Transport.onPacket(Transport.java:8)
       at io.socket.engineio.client.Transport.onData(Transport.java:2)
       at io.socket.engineio.client.transports.WebSocket.access$100(WebSocket.java:2)
       at io.socket.engineio.client.transports.WebSocket$1$2.run(WebSocket.java:2)
       at io.socket.thread.EventThread$2.run(EventThread.java:3)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at java.lang.Thread.run(Thread.java:929)
```      
## Information ##
```
OS Version : Android 10
Phones : HUAWEI P30 lite
SDK Version : 1.0.11
```

**Comment by Doc1faux** [comment]
Hi @kakarrot and thank your for your feedback.

I've just pushed an update which should fix this crash (and I hope all `null Context`...).
It will be released in the next `1.0.12`.
I let you know when it will be available.

**Comment by Doc1faux** [comment]
@kakarrot `1.0.12` fixing this issue has just been released.
It can take a few hours before it will be available.
I close the issue.
